pages/resources/lobby.mdx

```diff
@@ -0,0 +1,275 @@
+# Lobbies
+
+### Lobby Object
+
+Represents a lobby within Discord.
+
+###### Lobby Structure
+
+| Field           | Type                                                | Description                                       |
+| --------------- | --------------------------------------------------- | ------------------------------------------------- |
+| id              | snowflake                                           | The ID of the lobby                               |
+| application_id  | snowflake                                           | The ID of the application that created the lobby  |
+| metadata        | ?map[string, string]                                | The metadata of the lobby (max total length 1000) |
+| members         | array[[lobby member](#lobby-member-object) object]  | The members of the lobby                          |
+| linked_channel? | [channel](/resources/channel#channel-object) object | The guild channel linked to the lobby             |
+
+###### Example Lobby Object
+
+```json
+{
+  "id": "1350184413060661332",
+  "application_id": "891436243903728565",
+  "metadata": {
+    "topic": "balls"
+  },
+  "members": [
+    {
+      "id": "852892297661906993",
+      "metadata": null,
+      "flags": 1
+    }
+  ]
+}
+```
+
+### Lobby Member Object
+
+Represents a member of a lobby.
+
+###### Lobby Member Structure
+
+| Field             | Type                 | Description                                              |
+| ----------------- | -------------------- | -------------------------------------------------------- |
+| id                | snowflake            | The ID of the user                                       |
+| metadata?         | ?map[string, string] | The metadata of the lobby member (max total length 1000) |
+| flags?            | integer              | The [lobby member's flags](#lobby-member-flags)          |
+| connected ^1^ ^2^ | boolean              | Whether the member is connected to call in a lobby       |
+
+^1^ This field can be only received.
+
+^2^ This field is provided only if object was received over the Gateway.
+
+###### Lobby Member Flags
+
+| Value  | Name           | Description                                     |
+| ------ | -------------- | ----------------------------------------------- |
+| 1 << 0 | CAN_LINK_LOBBY | Lobby member can link a text channel to a lobby |
+
+### Lobby Message Object
+
+A message sent in a lobby within Discord.
+
+{/* TODO: Should this be documented as superset of a normal Message object? */}
```

Comment:
yes definitely. should also have a callout for the additional fields in ephemeral dm messages (recipient_id, recipients, channel (and the special channel object), all in the partial message structure